### ISSUE TITLE
Multipart and ExtractArchive Order

### ISSUE THREAD

**Issue by vertigo888** [pyload/pyload#3063]
Hi, I downloaded files from Mega.co.nz and files downloaded successfully. The files are several multipart 7z archives (001, 002, etc). Pyload successfully merges the archives but does not extract after merging - I receive "No files found to extract" after the merging has commenced.

Please help. Thanks!

**Comment by GammaC0de** [comment]
@vertigo888 You should disable the `MergeFiles` plugin.

**Comment by vertigo888** [comment]
I tied this. Turned off MergeFiles, restarted the download and disabled overwriting existing files (download already completed) and still same issue - archives didn't extract. I then went to extractarchive plugin and added 001 to list of extensions to monitor for extraction and activated the plugin to run only after all downloads are finished, still same issue. Any ideas? 

I downloaded p7zip and used this to extract, so archives extract properly and no corruption issues with the archives themselves.
